docs/ref/contrib/staticfiles.txt

```diff
@@ -230,6 +230,11 @@ Example usage::
 
 .. _staticfiles-storages:
```

Comment:
Please put your changes above this line, so internal doc links are still working.